sponnet/pipeline.libsonnet

```diff
@@ -73,15 +73,7 @@
         kind: matchArtifact.kind,
       },
     },
-    withDefaultArtifact(defaultArtifact):: self + {
-      defaultArtifact: {
-        reference: defaultArtifact.reference,
-        type: defaultArtifact.type,
-        kind: if defaultArtifact.kind == 'custom' then defaultArtifact else 'default.' + defaultArtifact.kind,
```

Comment:
I believe this value was incorrect because `kind` shouldn't have been an object when `defaultArtifact.kind == 'custom'` -- I'm guessing it should've been `"custom"` in that case.

I also suspect this field is not really used. At least it's not exported in the JSON document I get from Deck.